apps/website/src/components/content/LinkHover.tsx

```diff
@@ -0,0 +1,150 @@
+"use client";
```

Comment:
What is the purpose of this? We're not using client/server-specific components anywhere else in the stack?